source/_integrations/lookin.markdown

```diff
@@ -3,16 +3,18 @@ title: LOOKin
 description: Instructions on how to integrate LOOKin devices into Home Assistant.
 ha_category:
   - Climate
+  - Light
   - Media Player
   - Sensor
-ha_release: 2021.11
+ha_release: 2022.1.0
```

Comment:
```suggestion
ha_release: 2021.11
```

We don't ever change this as its referring to when the base integration was added to Home Assistant